### PR TITLE
Fix initial zoom/rotation + nightly

### PR DESCRIPTION
**Pull request by philippotto**
This PR fixes a weird initial rotation on the 3D viewport and also an incorrectly clamped zoom value. A slight difference in the default zoom value for the 3d viewport still exists, but I think it's not really harmful.

I created https://github.com/scalableminds/webknossos/issues/5455 as a follow-up to do a more thorough clean up on the initialization process.

### URL of deployed dev instance (used for testing):
- https://fixnightly.webknossos.xyz

### Steps to test:
- see CI run

### Issues:
- fixes nightly

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [(unreleased) migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes
- [ ] Needs datastore update after deployment
- [X] Ready for review

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -286,6 +286,9 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - nightly:
+          requires:
+            - build_test_deploy
```
Comment (philippotto) [inline .circleci/config.yml:291]: todo: undo before merging

File: frontend/javascripts/oxalis/controller/td_controller.js [hunk 2]
```diff
@@ -211,16 +212,16 @@ class TDController extends React.PureComponent<Props> {
     return controls;
   }
 
-  setTargetAndFixPosition(position?: Vector3): void {
+  setTargetAndFixPosition = (position?: Vector3): void => {
+    console.log("setTargetAndFixPosition");
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/controller/td_controller.js:216]:
```suggestion
```

File: frontend/javascripts/oxalis/controller/td_controller.js [hunk 3]
```diff
@@ -269,10 +270,14 @@ class TDController extends React.PureComponent<Props> {
   }
 
   render() {
+    // if (this.controls == null) {
+    //   return null;
+    // }
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/controller/td_controller.js:275]: Can this be removed?

File: frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js [hunk 4]
```diff
@@ -134,6 +134,9 @@ class PlaneController extends React.PureComponent<Props> {
     // catchers (only necessary for HammerJS). We should refactor the
     // InputMouse handling so that this is not necessary anymore.
     // See: https://github.com/scalableminds/webknossos/issues/3475
+
+    // const tdId = `inputcatcher_${OrthoViews.TDView}`;
+    // Utils.waitForElementWithId(tdId).then(() => {
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js:139]: Same, can this be removed?

**Review by daniel-wer — APPROVED** [review]
Glad that you were able to fix this for now, but I agree that the initialization procedure seems very messy and we should improve it sooner than later to avoid additional bugs. Thanks for creating the followup issue.
Ready to be merged once you addressed my small comments and reverted the changes in the config.yml :)

File: frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js (hunk 4 above)
Comment (philippotto) [reply frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js]: Whoops, I thought I had removed dead code, but apparently I was wrong. Should be cleaned up now!

**Pull request merged**